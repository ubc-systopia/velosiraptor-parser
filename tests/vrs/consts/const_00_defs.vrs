const FOO : int = 123;
const FOO : int = 0x1234;
const FOO : int = a;
const FOO : int = BAR;
const FOO : size = FOO + FOOBAR;
const FOO : MyType = 1234;
const FOO : bool = 1234;
const FOO : int = true;
