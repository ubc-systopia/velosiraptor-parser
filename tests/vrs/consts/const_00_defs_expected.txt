VelosiParseTree(tests/vrs/consts/const_00_defs.vrs)
---------------------------------------------
const FOO : int = 123;

const FOO : int = 4660;

const FOO : int = a;

const FOO : int = BAR;

const FOO : size = FOO + FOOBAR;

const FOO : MyType = 1234;

const FOO : bool = 1234;

const FOO : int = true;

---------------------------------------------
